test/models/passwordless/session_test.rb

```diff
@@ -47,5 +47,25 @@ def call(_session)
 
       Passwordless.token_generator = old_generator
     end
+
+    test 'with a custom expire at function' do
+      custom_expire_at = DateTime.parse('01-01-2100').utc
+      Passwordless.expires_at = lambda { custom_expire_at }
```

Comment:
These new settings will leak into the other tests. I think we should either …

1. Save `old_expire = Passwordless.expires_at` up front and then `Passwordless.expires_at = old_expire` at the end.
2. Or make a new `Passwordless::DEFAULTS` hash that we can reset all settings from after every test example.

Pick whatever you want.